CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,37 @@
+# Important
+
+The Paradise GitHub is not exempt from community and server rules, especially rules 0, 1, and 4. An inability to abide by the rules on the GitHub will result in disciplinary action up to, or including, a repository ban.
+
+## General Expectations
+
+### PR Discussion
+
+Comments on Pull Requests should remain relevant to the PR in question and not derail discussions.
+
+Under no circumstances are users to be attacked for their ideas or contributions. While constructive criticism is encouraged, toxicity or general mean-spirited behaviour will not be tolerated. All participants on a given PR or issue are expected to be civil. Failure to do so will result in disciplinary action.
+
+"Merge-nagging" or similar behaviour is not acceptable. Comments of this nature will result in warnings or an outright ban from the repository depending on general behaviour.
+
+If you exhibit behaviour that's considered to be a net-negative to the community (offensive commentary, repeat violations, constant nagging, personal attacks, etc.), you may be banned from other Paradise services (discord, forums, server, wiki, etc.)
+
+Maintainers reserve the right to permanently revoke access from the repository if your behaviour is considered to be a net negative.
+
+### PR Approval/Objection Info
+
+Heads of Staff and Maintainers have final say on Pull Requests. While thumbs ratios are generally taken into account, they do not dictate whether or not a PR will be merged.
+
+After a twenty four hour minimum waiting period, Pull Requests can be merged once they receive approval from both a Head of Staff and a Maintainer. An exception is made for refactors and fixes, which may be merged at a Maintainer's discretion with no waiting period.
+
+Heads of Staff and Maintainers are not obligated to publicly state their objections to a Pull Request. Attacking or berating either of these roles over an objection will not be tolerated. Additionally, whining over the closure of a PR, the existence of an objection, or similar behaviour, will not be tolerated.
+
+### PR Expectations
+
+All Pull Requests are expected to be tested prior to submission. If a submitted Pull Request fails to pass CI checks, the likelihood of it being merged will be significantly lower. If you can't take the time to compile/test your Pull Request, do not expect a warm reception.
```

Comment:
It means stuff like the basic compile test.

We wont penalise for the weird things dreamchecker picks up, such as
```dm
proc/thing()
```
instead of
```dm
/proc/thing()
```

However if the base code fails to even compile, thats grounds for lower merge chance.

Reply:
Could maybe be made clearer then, but it's probably fine as it is.